crow/timeline.md

```diff
@@ -5,26 +5,26 @@ nav_exclude: true
 
 # timeline
 
-`timeline` is all about sequencing events in time. i often noticed that i was writing out the same clock routines to get a basic rhythm going and i knew there was a better way! `timeline` is built on top of `clock`, so all the usual details for controlling tempo and clock-source apply here as well.
+`timeline` is all about sequencing events in time. I often noticed that I was writing out the same clock routines to get a basic rhythm going and I knew there was a better way! `timeline` is built on top of `clock`, so all the usual details for controlling tempo and clock-source apply here as well.
 
-the library has 3 flavours each with it's own purpose, though they can be combined in interesting ways too.
+The library has 3 flavours, each with it's own purpose, though they can be combined in interesting ways too.
 
 * `loop` is for (short) clock synchronized loops in terms of beat-durations.
 * `score` creates longer form sequences of events for song structure.
-* `real` is free from the clock, calling events over time. best for effects and real world time.
+* `real` is free from the clock, calling events over time, and is best for effects and real-world time.
 
-all the events can be launched quantized to the clock, and can all be repeated a programmatic number of times.
+All the events can be launched quantized to the clock, and can all be programmatically repeated a number of times.
 
 ## loop
 
-we'll start with a simple clock pulse, sent every beat on output 1:
+We'll start with a simple clock pulse, sent every beat on output 1:
 ```lua
 -- timeline version
 timeline.loop{1, function() output[1](pulse())}
 ```
-when we call the three core `timeline` functions, we always use curly braces. we're passing a table of elements to the `loop` function: first a time duration (as for `clock.sync`), then a function to be called. think of the time element as a *duration* of beats that the event runs over.
+When we call the three core `timeline` functions, we always use curly braces. We're passing a table of elements to the `loop` function: first a time duration (used by `clock.sync`), then a function to be called. Think of the time element as a *duration* of beats that the event runs over.
```

Comment:
@trentgill , since `clock.sync` doesn't technically work in durations (eg. `clock.sync(4)`will sync to the next-nearest multiple of 4, but that could be in a half-beat or 3 beats, depending on when it's called) , is there an alternate phrasing that can be used here?

Reply:
agreed, it's confusing. i guess i was just trying to communicate that it's in *beats* not seconds. open to any suggestions / edits here. we could even just drop everything in the parens.